README.md

```diff
@@ -74,6 +74,8 @@ Notes:
 - The `-DCMAKE_INSTALL_PREFIX=../release` option shown above is optional, and specifies the location where `nupic.core` should be installed. If omitted, `nupic.core` will be installed in a system location. Using this option is useful when testing versions of `nupic.core` with `nupic` (see [NuPIC's Dependency on nupic.core](https://github.com/numenta/nupic/wiki/NuPIC's-Dependency-on-nupic.core)).
 - Setting `-DPY_EXTENSIONS_DIR` copies the Python exension files to the specified directory. If the extensions aren't present when the Python build/installation is invoked then the setup.py file will run the cmake/make process to generate them. Make sure to include this flag if you want to do incremental builds of the Python extensions.
 - On OSX with multiple Python installs (e.g. via brew) cmake might erroneously pick various pieces from different installs which will likely produce abort trap at runtime. Remove cmake cache and re-run cmake with  `-DPYTHON_LIBRARY=/path/to/lib/libpython2.7.dylib` and  `-DPYTHON_INCLUDE_DIR=/path/to/include/python2.7` options to override with desired Python install path.
+- To use Include What You Use during compilation, pass '-DNUPIC-IWYU=ON'.  This requires that IWYU is installed and findable by CMake, and a minimum CMake version of 3.3.  IWYU can be installed on OS X using https://github.com/jasonmp85/homebrew-iwyu.
```

Comment:
A few nit picks:

- Don't use double spaces after periods
- To match surrounding commands, use back-ticks instead of single-quotes for command-line examples
- "with a minimum CMake version..."